apps/ima/translations/src/en/validation.json

```diff
@@ -1 +1,9 @@
-{}
+{
+    "image": {
```

Comment:
I think this needs to be formatted - too many spaces, will be picked up by linting

Reply:
Spaces formatted